### ISSUE TITLE
Quest: Hah...You're not so big now

### ISSUE THREAD

**Issue by Ergates82** [dalaranwow/dalaran-wow#5093]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Using Crafty's Ultra-Advanced Proto-Typical Shortening Blaster on a Magmoth Crusher or Mate of Magmothregar . Instead the mob sometimes applies the debuff, but the quest criteria is not updated. The game then crashes to desktop with an error message.

**Expected behaviour**: Tell us what should happen instead

Using Crafty's Blaster should apply one of several random debuffs to the mob - and also update the quest with a positive result, meaning that 1/5 of the mobs is now successfully tested etc. The game should not crash to desktop after attempting to use the blaster.

**Steps to reproduce the problem**:

1. Pick up the quest Hah...You're not so big now
2. Go to quest location 
3. Target either a Magmoth Crusher or Mate of Magmothregar,
4. Use Crafty's blaster.

**Include proofs for this behaviour**
http://www.wowhead.com/quest=11653/hah-youre-not-so-big-now

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Dup with #706
